lib/index.ts

```diff
@@ -68,11 +98,13 @@ class ProtractorScreenShotUtils {
             element:ElementFinder = options.element ? options.element : currentContext.$("body"),
             dimensions = options.dimensions || {},
             outputPath = options.saveTo || null,
+            canvasOptions = options.canvasOptions || {},
```

Comment:
how about we include the canvasOptions in the constructor and use it as a default open. we can also override the default options when calling the takeScreenshot method?